compiler/scopelang/ast.mli

```diff
@@ -58,6 +58,7 @@ type 'm scope_decl = {
   scope_sig : scope_var_ty ScopeVar.Map.t;
   scope_decl_rules : 'm rule list;
   scope_options : Desugared.Ast.catala_option Mark.pos list;
+  scope_visibility : visibility;
```

Comment:
Is this visibility related to the change I propose in https://github.com/CatalaLang/catala/pull/624/commits/ecc1c77aa38b9b245e0c6cada7cf46202f15ea0a ?

Reply:
Not directly, it's not related to the I/O, but to the fact that it's exposed in the interface of the module (type visibility is `Public | Private`); which corresponds to the definition being in a `catala-metadata` or plain `catala` block. I needed this in particular to ensure stable renaming of public parts of a module, independently of the private parts.